ckanext/validation/assets/js/validation-badge.js

```diff
@@ -0,0 +1,55 @@
+this.ckan.module('validation-badge', function (jQuery) {
+    return{
+        options: {
+          resource: null,
+          status: 'created',
+          url: null
+        },
+        initialize: function () {
+          $.proxyAll(this, /_on/);
+          this._poll();
+          this.options.url = $(this.el).attr('href');
+          $(this.el).removeAttr('href');
+        },
+        _poll: function() {
+            var module = this;
+            $.ajax({
+                url: "/api/3/action/resource_validation_show?resource_id=" + this.options.resource,
```

Comment:
Is it possible to use some sort of `url_for` here not to hard code the API path?
In HTML Jinja we use e.g. `{% url_for 'dataset.search', tags=tag.name %}`

Reply:
As stated earlier, documentation is scarce and studying the source code suggests there is not clear way to do this.